src/scripts/pages/about/contact/contact-template.html

```diff
@@ -15,7 +15,7 @@
 
   <p>
     <strong>Phone:</strong> 713-348-5012<br />
-    <strong>Email:</strong> cnx@cnx.org
+    <strong>Email:</strong> support@openstax.org
```

Comment:
My thoughts are make the minimum possible change that  can't possibly break, then change it correctly (make this a config property, probably) on master. Would making it a mailto: fit the first or the second?